compiler/rustc_index/src/bit_set.rs

```diff
@@ -590,13 +590,13 @@ impl<T: Idx> SparseBitSet<T> {
 
     fn contains(&self, elem: T) -> bool {
         assert!(elem.index() < self.domain_size);
-        self.elems.contains(&elem)
+        self.elems.iter().any(|e| elem.index() == e.index())
```

Comment:
Are these changes necessary? `Eq` is still implemented, right?

Reply:
Ah! Let me check. I think there were some changes need as I was working to keep the build working, but not after completing certain changes. (This must be a common experience...)